third_party/terraform/resources/resource_iam_binding.go

```diff
@@ -57,11 +57,8 @@ func resourceIamBindingCreate(newUpdaterFunc newResourceIamUpdaterFunc) schema.C
 
 		p := getResourceIamBinding(d)
 		err = iamPolicyReadModifyWrite(updater, func(ep *cloudresourcemanager.Policy) error {
-			// Creating a binding does not remove existing members if they are not in the provided members list.
-			// This prevents removing existing permission without the user's knowledge.
-			// Instead, a diff is shown in that case after creation. Subsequent calls to update will remove any
-			// existing members not present in the provided list.
-			ep.Bindings = mergeBindings(append(ep.Bindings, p))
+			// Creating a binding will overwrite the existing list of members for that role.
+			ep.Bindings = overwriteBinding(ep.Bindings, p)
```

Comment:
`Update` is already authoritative; can we use a unified `CreateUpdate` method instead? We've done that in a few other places such as https://github.com/terraform-providers/terraform-provider-google/blob/master/google/resource_storage_default_object_acl.go#L11 for similar-ish resources.